.github/workflows/manual.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  schedule:
+    - cron:  '0 7 * * 1,7'
+    # scheduled at 07:00 every Monday and Thursday
```

Comment:
Please, do not commit this file.

(Note: if you want to use this workflow **on your fork**, consider that there is a typo here. Week day _7_ is [non-standard](https://crontab.guru/), and, even if it works, it would reference Sunday, not Thursday as the comment days).